src/app/modules/feedback/feedback.util.ts

```diff
@@ -0,0 +1,63 @@
+import { StatusCodes } from 'http-status-codes'
+
+import { MapRouteError } from '../../../types'
+import { createLoggerWithLabel } from '../../config/logger'
+import { ApplicationError, DatabaseError } from '../core/core.errors'
+import * as FormErrors from '../form/form.errors'
+
+import {
+  DuplicateFeedbackSubmissionError,
+  InvalidSubmissionIdError,
+} from './feedback.error'
+
+const logger = createLoggerWithLabel(module)
+
+export const mapRouteError: MapRouteError = (
+  error: ApplicationError,
+  coreErrorMessage?: string,
+) => {
+  switch (error.constructor) {
+    case FormErrors.FormNotFoundError:
+      return {
+        statusCode: StatusCodes.NOT_FOUND,
+        errorMessage: error.message,
```

Comment:
can we go through each error message to ensure that it makes sense to the user in the context of form feedback? e.g. in this case it's just "Form not found". if a user submits the form, and while they're typing feedback the form admin makes it private, then the user tries to submit the feedback, they will see "Form not found" and be confused. maybe we can make this error message specific to this context, e.g. "This form has been made private, so feedback submissions are no longer accepted."

Reply:
Updated the PR for clearer error messages in the context of feedback, thanks!